modules/@apostrophecms/piece-type/ui/apos/components/AposImageCropper.vue

```diff
@@ -59,42 +83,268 @@ export default {
     }
   },
   created () {
-    this.setCoordinates = debounce(
-      (coordinates) => {
-        this.$refs.cropper.setCoordinates(coordinates);
-      }, 500
-    );
+    this.setCropperCoordinates = debounce(coordinates => {
```

Comment:
Please change these to follow your pattern below in which debounced versions of methods are made, but the actual method code is still in "methods."